spring-data-geode-test/src/main/java/org/springframework/data/gemfire/tests/integration/ForkingClientServerIntegrationTestsSupport.java

```diff
@@ -15,26 +15,24 @@
  */
 package org.springframework.data.gemfire.tests.integration;
 
-import static org.springframework.data.gemfire.util.ArrayUtils.nullSafeArray;
```

Comment:
Please configure your IDE to leave the imports as is (e.g. order, styling, etc).